### PR TITLE
[OFFAPPS-702] updated translations

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @injuhwang 

### Description

Updated translations

### References
* JIRA: https://zendesk.atlassian.net/browse/OFFAPPS-702

### Risks
None

### CODE REVIEW

File: translations/ar.json [hunk 1]
```diff
@@ -76,6 +80,7 @@
       "solution": "إن لم تفعل، فراجع <a href=\"{{url}}\" target=\"_blank\">هذه التعليمات</a> وانتقل إلى صفحة إدارة نماذج التذاكر لإتمام الإعداد."
     },
     "save_hook": "يجب عليك إرسال الوقت الذي أنفقته على هذه التذكرة.",
-    "bad_time_format": "تنسيق وقت غير صالح (س‏س:دد:ث‏ث)."
+    "bad_time_format": "تنسيق وقت غير صالح (س‏س:دد:ث‏ث).",
+    "negative_time": "كان الوقت سالبًا. سيُعاد ضبط الوقت إلى الصفر."
```
Comment (sandlerr) [inline translations/ar.json]: @ocke this string is obsolete right? we shouldn't waste any more translators time on it, can you flag it as obsolete in the yml please?

**Comment by sandlerr** [comment]
why do we have `uk` and `en-gb`? :+1:

**Comment by svizzari** [comment]
👍

**Comment by injuhwang** [comment]
Please don't forget Bulgarian. 😉

**Comment by pmgrove** [comment]
@injuhwang, apologies, added Bulgarian.

**Comment by injuhwang** [comment]
👍 Thanks a lot!

**Pull request merged**